pkgs/servers/code-server/default.nix

```diff
@@ -98,17 +102,16 @@ in stdenv.mkDerivation rec {
     grep -rl "yarn install" --include package.json lib/vscode/extensions \
       | xargs sed -i 's/yarn install/yarn install --offline/g'
 
+    substituteInPlace ci/dev/postinstall.sh \
+      --replace 'yarn' 'yarn --ignore-scripts'
+
     # remove download of coder-cloud agent
-    sed -i ':a;N;$!ba;s/OS=.*agent//' ci/build/npm-postinstall.sh
+    sed -i ':a;N;$!ba;s/\n  OS=.*work"\n  fi\n//' ci/build/npm-postinstall.sh
```

Comment:
This looks like a good candidate for a patch.

Reply:
👍 I'll add a patch for this